### PR TITLE
[VPU]  Added support for 2 axis for MVN layer - duplicate

### PR DESCRIPTION
**Pull request by DariaMityagina**
### Details:
 - This pr adds support for Person-reidentification-retail model on VPU device by adding support for {2} axis in MVN layer

### Tickets:
 - 55666

duplicates https://github.com/openvinotoolkit/openvino/pull/6669

### CODE REVIEW

**Comment by DariaMityagina** [comment]
@Maxim-Doronin @AntonDudchenko could you please take a look? The original PR already has approves, need them here as well. 
Validation is fine.

File: inference-engine/cmake/vpu_dependencies.cmake [hunk 1]
```diff
@@ -6,14 +6,14 @@ include_guard(GLOBAL)
...
-    "61797a77b38fc677be4cc63d730e8871bbf169686b88eabb7066b01f9d156129")
+    "0731d136349c3ad7e5de91987ee5d721f392683cbc53e3b7e7e1e200cf1eb3d0"
+    "fa97ce2fb7cc6f392af1946ae393b40c7bada4c355d38fd7ad4a92ad554b5a3e")
 
 #
 # Default packages
 #
 
-set(FIRMWARE_PACKAGE_VERSION 1714)
+set(FIRMWARE_PACKAGE_VERSION 1712)
```
Comment (Maxim-Doronin) [inline inference-engine/cmake/vpu_dependencies.cmake:16]: Version is being downgraded? interesting

File: inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp [hunk 2]
```diff
@@ -88,11 +90,13 @@ void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& layer, const
     for (int i = 0; i < indicesSize; i++) {
         axes.insert(getDimFromAxis(ndims, indicesPtr[i]));
     }
+    const auto width = axes.count(Dim::W);
 
-    VPU_THROW_UNLESS(!axes.count(Dim::N) && axes.count(Dim::H) && axes.count(Dim::W),
+    VPU_THROW_UNLESS(!axes.count(Dim::N) && width,
                      "Unsupported combination of indices in layer \"%s\". "
                      "Only across channel and full batch supported.", layer->name);
```
Comment (Maxim-Doronin) [inline inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp:97]: please, fix error message according to the changes

File: inference-engine/cmake/vpu_dependencies.cmake (hunk 1 above)
Comment (DariaMityagina) [reply inference-engine/cmake/vpu_dependencies.cmake:16]: My bad. Will be fixed. Thanks

Comment (DariaMityagina) [reply inference-engine/cmake/vpu_dependencies.cmake]: fixed

File: inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp (hunk 2 above)
Comment (DariaMityagina) [reply inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp]: fixed

**Comment by elatkin** [comment]
@DariaMityagina 
Do you plan to close the original #6669?

**Comment by DariaMityagina** [comment]
> @DariaMityagina
> Do you plan to close the original #6669?

Yes, sure

**Review by elatkin — APPROVED** [review]

**Review by Maxim-Doronin — APPROVED** [review]

**Pull request merged**